integration-tests/generate_from_templates.py

```diff
@@ -0,0 +1,108 @@
+parser.add_argument('--create-pearl-wait-delay', dest='create_pearl_wait_delay', type=str, default='100ms', help='each thread will wait this period if another thread creating pearl.')
+parser.add_argument('--metrics-name', dest='metrics_name', type=str, default='bob', help='add base name for metrics.')
+parser.add_argument('--graphite-enabled', dest='graphite_enabled', type=str, default='true', choices=['true', 'false'])
+parser.add_argument('--prometheus-enabled', dest='prometheus_enabled', type=str, default='false', choices=['true', 'false'])
+parser.add_argument('--path', dest='path', type=str, help='sets path to directory where configs will be generated.', default='/tmp')
+
+args = parser.parse_args()
+
+path = pathified(args.path) + '/generated_configs'
+
+os.makedirs(path, exist_ok=True, mode=0o777)
+
+for filename in os.listdir(path):
+    try:
+        os.remove(f"{path}/{filename}")
+    except OSError:
+        shutil.rmtree(f"{path}/{filename}")
+
+try:
+    original_umask = os.umask(0)
+    for i in range(args.amount_of_nodes):
+        os.makedirs(path + f'/data{i}/d1', exist_ok=True, mode=0o777)
+finally:
+    os.umask(original_umask)
+
+
+#generate compose file
+compose = open("Templates/compose_template.yml.j2").read()
+template = Template(compose)
+f = open(f"{path}/docker-compose.yml", 'w')
+f.write(template.render(amount_of_nodes=args.amount_of_nodes, version=args.version, path=path))
+f.close
+
+#generate node files
+for item in range(args.amount_of_nodes):
+    node = open("Templates/node_template.yml.j2").read()
+    template = Template(node)
+    f = open(f"{path}/node.yaml.bobnet{item}", 'w')
+    f.write(template.render(node_number=item, version=args.version, log_config=args.log_config, users_config=args.users_config, quorum=args.quorum, 
+    operation_timeout=args.operation_timeout, check_interval=args.check_interval, cluster_policy=args.cluster_policy, 
+    backend_type=args.backend_type, cleanup_interval=args.cleanup_interval, open_blobs_soft_limit=args.open_blobs_soft_limit,
+    open_blobs_hard_limit=args.open_blobs_hard_limit, bloom_filter_memory_limit=args.bloom_filter_memory_limit, 
+    auth_type=args.auth_type, index_memory_limit=args.index_memory_limit, enable_aio=args.enable_aio, 
+    disks_events_logfile=args.disks_events_logfile, max_blob_size=args.max_blob_size, allow_duplicates=args.allow_duplicates, 
+    max_data_in_blob=args.max_data_in_blob, blob_file_name_prefix=args.blob_file_name_prefix,
+    fail_retry_timeout=args.fail_retry_timeout, alien_disk=args.alien_disk, 
+    bloom_filter_max_buf_bits_count=args.bloom_filter_max_buf_bits_count, root_dir_name=args.root_dir_name, 
+    alien_root_dir_name=args.alien_root_dir_name, timestamp_period=args.timestamp_period,
+    create_pearl_wait_delay=args.create_pearl_wait_delay, metrics_name=args.metrics_name, graphite_enabled=args.graphite_enabled,
+    prometheus_enabled=args.prometheus_enabled))
```

Comment:
This will be hard to maintain. Please, find out how to translate all arguments into `render` parameters without listing them all explicitly